lumicks/pylake/kymo.py

```diff
@@ -570,6 +572,74 @@ def plot_with_time_histogram(
         ax_hist.set_ylabel("counts")
         ax_hist.set_title(self.name)
 
+    def bead_roi(self, bead_diameter, *, channel="green", plot=False, percentile=70, **kwargs):
+        """Determine approximate positional coordinates of the bead edges.
+
+        Searches for the bead edges by checking where the fluorescence drops below a threshold.
+        Only intended for non-moving beads.
+
+        .. warning::
+
+            This is early access alpha functionality. While usable, this has not yet been tested in
+            a large number of different scenarios. The API can still be subject to change without
+            any prior deprecation notice! If you use this functionality keep a close eye on the
+            changelog for any changes that may affect your analysis.
+
+        Parameters
+        ----------
+        bead_diameter : float
+            Rough estimate for the bead size (microns).
+        channel : 'red', 'green', 'blue', optional
+            Channel to use for bead detection.
```

Comment:
Not for this PR, but just curious did you try this by summing across channels also? Often the beads show up in all channels and using all the signal would significantly boost the beads features even more above anything on the tether

Reply:
Great idea! I did just give this a go, but for the kymos that I have it doesn't seem to make a big difference. If you have one where you see a big difference in SNR that way, it would be good to include it in the test set.

I still have a few failure mode kymos. Basically, if the center gets really crowded or there are mobile proteins, the small peak rejection fails. Usually this can be recovered by going for a different channel or taking a subset of the kymo.

Reply:
I'm not so worried about failure modes because the beads can be really variable so I'm not expecting 100% accuracy for this estimate. I just wonder, if using the signal summed across channels isn't any worse off than using a single channel maybe we just do this by default and not have the channel argument at all?